api/src/routes/index.test.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint @typescript-eslint/no-var-requires: "off"
```

Comment:
I don't think this annotation is needed here. Am I missing something?

Reply:
made change